tools/appveyor_install.ps1

```diff
@@ -0,0 +1,96 @@
+# Sample script to install Miniconda under Windows
```

Comment:
I use "ci" which I found easier to identify than "integration" (It's also used be pandas)...